bin/docker/boot_dev

```diff
@@ -107,16 +107,17 @@ docker run -d \
     discourse/discourse_dev:release /sbin/boot
 
 if [ "${initialize}" = "initialize" ]; then
-    echo "Installing gems..."
-    "${SCRIPTPATH}/bundle" install
-
-    echo "Yarn install..."
-    "${SCRIPTPATH}/exec" yarn install --cwd app/assets/javascripts/discourse
-
     echo "Migrating database..."
     "${SCRIPTPATH}/rake" db:migrate
     RAILS_ENV=test "${SCRIPTPATH}/rake" db:migrate
```

Comment:
I think it might make sense for migrations to be run manually because one might be in the midst of adding a new migration and does not want to run it yet.